### PR TITLE
Adds Durations to CI tests

### PR DESCRIPTION
**Pull request by badisa**
* Makes easier to determine where time is spent

Top couple from CI Run:
```
319.21s call     tests/test_md.py::test_reference_langevin_integrator_with_custom_ops
206.10s call     tests/test_rabfe.py::TestRABFEModels::test_predict_complex_decouple
180.40s call     tests/test_rabfe.py::TestRABFEModels::test_predict_complex_conversion
120.34s call     tests/test_md.py::test_reference_langevin_integrator
117.82s call     tests/test_nonbonded.py::TestNonbonded::test_nonbonded
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 MKFILE_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 CPP_DIR := $(MKFILE_DIR)timemachine/cpp/
 INSTALL_PREFIX := $(MKFILE_DIR)timemachine/
-PYTEST_CI_ARGS := --cov=. --cov-report=term-missing
+PYTEST_CI_ARGS := --cov=. --cov-report=term-missing --durations=100
```
Comment (badisa) [inline Makefile:5]: Picked 100 as it gives a good number of timings, but doesn't grow with more tests. Let me know if there is a preferred value.

Comment (maxentile) [reply Makefile:5]: Looks good to me-- also durations < 0.005s hidden by default

**Comment by maxentile** [comment]
Nice!

qq: there are two groups of durations in the log, separated by a few hundred lines: are these `slow_tests/` and `tests/`?

**Review by maxentile — APPROVED** [review]

**Comment by badisa** [comment]
> Nice!
> 
> qq: there are two groups of durations in the log, separated by a few hundred lines: are these `slow_tests/` and `tests/`?

Correct, they are broken up into the separate runs.

**Review by mcwitt — APPROVED** [review]

**Review by proteneer — APPROVED** [review]
lgtm

**Pull request merged**